src/lang/en-US.json

```diff
@@ -24,7 +24,36 @@
 
   "announce_dismiss": "Dismiss",
   "announce_github_more": "View on GitHub",
-
+  
+  "api_title": "The OpenDota API",
+  "api_subtitle": "Build on top of the OpenDota platform. Bring advanced stats to your app and deep insights to your users.",
+  "api_first": "First",
+  "api_calls_month": "calls per month",
+  "api_free": "Free",
+  "api_per_unit": "per 1000 calls",
+  "api_get_key": "Get my key",
+  "api_docs": "Read the docs",
+  "api_header_details": "Details",
+  "api_key_requirement": "Calls over 25000/month will require an API key.",
```

Comment:
discussed with some of the current API consumers and 50,000 may be a better free tier.  You can get their opinions on Discord